prowler/lib/outputs/jira/jira.py

```diff
@@ -0,0 +1,1040 @@
+                    logger.error("No projects found")
+                    raise JiraNoProjectsError(
+                        message="No projects found in Jira",
+                        file=os.path.basename(__file__),
+                    )
+                issue_types = response.json()["projects"][0]["issuetypes"]
+                return [issue_type["name"] for issue_type in issue_types]
+            else:
+                # Must be replaced with proper error handling from custom exceptions
+                response_error = f"Failed to get available issue types: {response.status_code} - {response.json()}"
+                logger.warning(response_error)
+                raise JiraGetAvailableIssueTypesResponseError(
+                    message=response_error, file=os.path.basename(__file__)
+                )
+        except JiraRefreshTokenError as refresh_error:
+            raise refresh_error
+        except JiraRefreshTokenResponseError as response_error:
+            raise response_error
+        except Exception as e:
+            logger.error(f"Failed to get available issue types: {e}")
+            raise JiraGetAvailableIssueTypesError(
+                message="Failed to get available issue types",
+                file=os.path.basename(__file__),
+            )
+
+    def send_findings(
+        self,
+        findings: list[Finding] = None,
+        project_key: str = None,
+        issue_type: str = "Bug",
+    ):
+        """
+        Send the findings to Jira
+
+        Args:
+            - findings: The findings to send
+            - project_key: The project key
+            - issue_type: The issue type
+
+        Raises:
+            - JiraRefreshTokenError: Failed to refresh the access token
+            - JiraRefreshTokenResponseError: Failed to refresh the access token, response code did not match 200
+            - JiraCreateIssueError: Failed to create an issue in Jira
+            - JiraSendFindingsResponseError: Failed to send the findings to Jira
+        """
+        try:
+            access_token = self.get_access_token()
+
+            if not access_token:
+                return ValueError("Failed to get access token")
+
+            available_issue_types = self.get_available_issue_types(project_key)
+
+            if issue_type not in available_issue_types:
+                logger.error("The issue type is invalid")
+                raise JiraInvalidIssueTypeError(
+                    message="The issue type is invalid", file=os.path.basename(__file__)
+                )
+            headers = {
+                "Authorization": f"Bearer {access_token}",
+                "Content-Type": "application/json",
+            }
+
+            for finding in findings:
+                if finding.status.value == "PASS":
+                    status_color = "#008000"
+                else:
+                    status_color = "#FF0000"
+
+                adf_description = {
```

Comment:
I'd create another function with several parameters to customize the issue content, what do you think?

Reply:
Done